.buildkite/steps/publish-docker-image.sh

```diff
@@ -61,6 +61,14 @@ if [[ "$codename" == "stable" ]] ; then
   done
   release_image "${variant}"
 
+  # publish bare 'ubuntu' only from ubuntu-20.04
+  if [[ "$variant" == "ubuntu-20.04" ]] ; then
```

Comment:
I reckon duplicating the the loop above here makes more sense than embedding an `if` statement in the existing loop and again where we do the 'bare' variant release. This way there is only a single branch statement to update when we want to update the `ubuntu` tag series to point to a 22.04 release.